torch/_inductor/codegen/cpp_prefix.h

```diff
@@ -484,6 +484,18 @@ inline at::vec::Vectorized<float> vec_shuffle_down(at::vec::Vectorized<float> x,
   }
   throw std::runtime_error("Unhandled vec_shuffle_down value " + std::to_string(n));
 }
+
+// For WOQ INT4
+inline __m128i convert_int4_to_int8(const uint8_t* data) {
```

Comment:
Moving this to aten vec is better?

Reply:
Since aten vec headers deal with `Vectorized` structs, rather than `__m128i`, I have moved this to microkernel code.